lib/galaxy/tools/parameters/wrapped_json.py

```diff
@@ -66,7 +66,11 @@ def _data_input_to_path(v):
             if value:
                 if isinstance(value, list):
                     value = value[0]
-                return _hda_to_object(value)
+                json_value = _hda_to_object(value)
+                if input.load_contents:
+                    with open(str(value), mode='rb') as fh:
+                        json_value['contents'] = fh.read(input.load_contents).decode('utf-8', errors='replace')
```

Comment:
Should we throw an error if the file is too big or something? This will occasionally crash Galaxy processes.

Reply:
Nvm, that is bytes not a boolean - very cool.